docs/release-notes/release-notes-0.15.1.md

```diff
@@ -69,6 +69,10 @@
   flow](https://github.com/lightningnetwork/lnd/pull/6754) so that the server 
   can indicate to the client that it has completed the RPC MW registration.
 
+* Add [TrackPayments](https://github.com/lightningnetwork/lnd/pull/6335)
```

Comment:
As pointed out by @guggero, we could aim this for `v0.16.0`.

Reply:
Moved to release notes of 0.16.0